account_multiple_account_type_reconciliation/static/src/js/account_multiple_account_type_reconciliation.js

```diff
@@ -0,0 +1,32 @@
+//-*- coding: utf-8 -*-
+//© 2017 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+openerp.account_multiple_account_type_reconciliation = function(instance) {
+	instance.web.account.bankStatementReconciliationLine.include({
+
+		selectMoveLine : function(mv_line) {
```

Comment:
When you try to reconcile bank statements that belong to a payable account and to a receivable account (debit, credit) you get the warning (web.Dialog) on `account/static/src/js/account_widgets.js`  at around line 1057. This module basically removes this warning by overriding the `selectMoveLine` method.